Automattic/jetpack
# Slideshow: Add unit tests and fixtures to validate block content parsing

#### Changes proposed in this Pull Request:
* Adds basic unit tests and block fixture parsing tests to the Slideshow block which currently has no test coverage.

#### Jetpack product discussion
See p1HpG7-b3G-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Check out the PR and check that insertion of Slideshow block still works as expected along with all setting of config via toolbar and side panel
* Run `yarn jest extensions/blocks/slideshow/test/controls.js` & `yarn jest extensions/blocks/slideshow/test/edit.js` & `yarn jest extensions/blocks/slideshow/test/validate.js` in jetpack dir and make sure they all pass 

#### Proposed changelog entry for your changes:
* No changelog entry needed
